RadarSDK/RadarAPIClient.m

```diff
@@ -37,6 +37,19 @@ - (instancetype)init {
     return self;
 }
 
++ (NSDictionary *)headersWithPublishableKey:(NSString *)publishableKey {
+    return @{
+        @"Authorization": publishableKey,
+        @"Content-Type": @"application/json",
+        @"X-Radar-Config": @"true",
```

Comment:
do we always want to return back the client's config in the SDK?

I know we use this for internal testing / confirmation, but not sure we need to expose that to customers

Reply:
The config bundle isn't being used on the client right now, but no issues retrieving it with each request